src/Commands/RestoreCommand.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/*
+ * This file is part of the hyn/multi-tenant package.
+ *
+ * (c) Daniël Klabbers <daniel@klabbers.email>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ *
+ * @see https://laravel-tenancy.com
+ * @see https://github.com/hyn/multi-tenant
+ */
+
+namespace Hyn\Tenancy\Commands;
+
+use Hyn\Tenancy\Models\Website;
+use Illuminate\Console\Command;
+use Hyn\Tenancy\Traits\DispatchesEvents;
+use Hyn\Tenancy\Events\Websites as Events;
+
+class RestoreCommand extends Command
+{
+    use DispatchesEvents;
+
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'tenancy:restore';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Restore all tenant databases present in the system db.';
+
+    /**
+     * Create a new command instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        parent::__construct();
```

Comment:
unnecessary construct, drop the construct method all together so it falls through to the parent without calling it explicitly

Reply:
hmmm. this is an output of make:command. I'll remove it.

Reply:
Fix the above and I think we can merge it as is. Make sure to remove the TODO comments too for now, no need to keep stuff like that in versioned code.